server/graphql/v2/query/collection/TransactionsCollectionQuery.ts

```diff
@@ -280,9 +290,17 @@ export const TransactionsCollectionResolver = async (
   if (args.dateTo) {
     where.push({ createdAt: { [Op.lte]: args.dateTo } });
   }
+  if (args.expense) {
+    const expenseId = getDatabaseIdFromExpenseReference(args.expense);
+    where.push({ ExpenseId: expenseId });
+  }
   if (args.hasExpense !== undefined) {
     where.push({ ExpenseId: { [args.hasExpense ? Op.ne : Op.eq]: null } });
   }
+  if (args.order) {
+    const orderId = getDatabaseIdFromOrderReference(args.order);
+    where.push({ OrderId: orderId });
```

Comment:
What if `args.order` is bogus and `orderId` end up being `null`.

Is it gonna return all entries that have `OrderId=null` ?

Reply:
@znarf it is handled by `getDatabaseIdFromOrderReference` that throws errors for invalid or missing id's. There seems to be no case to end up with returning entries that have `OrderId=null`